src/components/ResourceCard.astro

```diff
@@ -0,0 +1,39 @@
+---
+export type Props = {
+	class?: string
+	title: string
+	description: string
+	link: {
+		href: string
+		text: string
+	}
+}
+---
+
+<div class="h-full @container">
```

Comment:
Is this something we can do *well enough* without container queries here?

Last I checked I think support was still ~85%-90%, might be worth letting that feature age a little more unless the Tailwind plugin has some fancy fallbacks built in

Reply:
There's no fallback. I used it in this case because the UI still looks fine without them (top-to-bottom, everything centered) but a majority of viewers will get the designed UI